docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -0,0 +1,170 @@
+| Normalized tabular data 	         | Tabular Source (database, file, etc.) 	| `_airbyte` metadata columns, SCD tables                  	                     |
+
+For `destination-bigquery`, we are now loading `JSON` blobs as type `JSON` in BigQuery (introduced last [year](https://cloud.google.com/blog/products/data-analytics/bigquery-now-natively-supports-semi-structured-data)), instead of type `string`.
+
+
+## Quick Start to Upgrading
+
+The quickest path to upgrading is to click upgrade on any out-of-date connection in the UI:
+
+![Upgrade Path](./assets/airbyte_destinations_v2_upgrade_prompt.png)
+
+After upgrading the out-of-date destination to a [Destinations V2 compatible version](#destinations-v2-effective-versions), the following will occur at the next sync **for each connection** sending data to the updated destination:
+1. Existing raw tables replicated to this destination will be copied to a new `airbyte` schema. 
+2. The copied raw table will be updated to the new Destinations V2 format.
+3. The new raw tables will be updated with any new data since the last sync.
+4. The updated raw tables will be typed and de-duplicated according to the Destinations V2 format.
+5. Once typing and de-duplication has completed successfully, your previous final table will be replaced with the updated data.
+
+Pre-existing raw tables, SCD tables and "unnested" tables will always be left untouched. You can delete these at your convenience.
+Each destination version is managed separately, so if you have multiple destinations, they all need to be upgraded one by one.
+
+Versions are tied to the destination. When you update the destination, **all connections tied to that destination will be sending data in the Destinations V2 format**. For upgrade paths that will minimize disruption to existing dashboards, see:
+* [Upgrading Connections One by One with Dual-Writing](#upgrading-connections-one-by-one-with-dual-writing)
+* [Testing Destinations V2 on a Single Connection](#testing-destinations-v2-for-a-single-connection)
+* [Upgrading as a User of Raw Tables](#upgrading-as-a-user-of-raw-tables)
+* [(OSS Only) Rolling back to Legacy Normalization](#oss-only-rolling-back-to-legacy-normalization)
+ 
+## Advanced Upgrade Paths
+
+### Upgrading Connections One by One with Dual-Writing
+
+Dual writing is a method employed during upgrades where new incoming data is written simultaneously to both the old and new systems, facilitating a smooth transition between versions. We recommend this approach for connections where you are especially worried about breaking changes or downtime in downstream systems.
+
+#### Steps to Follow for All Sync Modes
+
+1. **[Open Source]** Update the default destination version for your workspace to a [Destinations V2 compatible version](#destinations-v2-effective-versions). This sets the default version for any newly created destination.
```

Comment:
```suggestion
1. **[Open Source]** Update the default destination version for your workspace to a [Destinations V2 compatible version](#destinations-v2-effective-versions). This sets the default version for any newly created destination.  Your existing syncs will remain on the previous version 
```

* assuming the OSS deployment has opted into breaking change management.  @erohmensing is working though this now